### DISCUSSION TITLE
longtblr and fancyhdr

### DISCUSSION THREAD

**Discussion by s163555** [lvjr/tabularray#346] [Q&A]
Hi everyone. When using a longtblr that extends multiple pages, the chapter mark does not extend into the subsequent pages. This is shown in the screenshot below:
![image](https://user-images.githubusercontent.com/72868007/210927583-d3703638-94e1-4999-bb3d-e786746ef954.png)
Is this due to tabularray or fancyhdr? If it is a matter for tabularray, is it possible to mark subsequent pages as well?
Thank you

**Comment by muzimuzhi** [comment]
Is the chapter mark expected to show only on odd page footers, or both odd and even page footers?

**Reply by s163555** [reply]
> Is the chapter mark expected to show only on odd page footers, or both odd and even page footers?

It is expected to show on both odd and even. In the preamble setup the two are identical. The table is 3 pages long and goes odd->even->odd and only shows up on the 1st (odd) page

**Reply by muzimuzhi** [reply]
Then please provide a complete example.

**Reply by muzimuzhi** [reply]
> Then please provide a complete example.

Partly because a simple example can't reproduce your problem:
```tex
\documentclass{article}
\usepackage[margin=1cm, noheadfoot, headsep=1cm, papersize={10cm,5cm}]{geometry}
\usepackage{tabularray}
\usepackage{fancyhdr}
\pagestyle{fancy}

\begin{document}
\markright{righthead}
\begin{longtblr}{
  hlines, vlines,
  colspec={X[1]X[2]},
}
  a & a \\ \pagebreak
  b & b \\
  c & c \\ \pagebreak
  d & d \\
  e & e \\
  f & f \\
\end{longtblr}
\end{document}
```
![image](https://user-images.githubusercontent.com/6376638/210943419-3de5e9c4-8852-4ec0-bf8a-038259adf243.png)

**Reply by s163555** [reply]
> > Then please provide a complete example.
> 
> Partly because a simple example can't reproduce your problem:
> ...
>

You're right! I am sorry, I happened to have put a \thispagestyle with a different type during troubleshooting that must have been affecting the subsequent pages. When I create an MWE from scratch, the fancyhdr is actually displaying correctly.
